CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ This changelog is a work in progress and may contain notes for versions which ha
 
 ### Bug fixes 🐞
 
+- Fixed a typo ("rendervouz" --> "rendezvous") in GetStatsResponse. ([#611](https://github.com/0x-mesh/pull/611)).
```

Comment:
@jalextowle this isn't the right place to add the entry to the changelog. `v8.0.0-beta-0xv3` has already been released. You need to add a heading for a new version called `v8.0.1` (without the `beta` or `0xv3` suffix this time).